src/App.js

```diff
@@ -1,12 +1,22 @@
 import React from "react";
 import "./App.css";
+import { Route, Routes } from "react-router-dom";
 import Header from "./components/Header/Header";
+import Homepage from "./components/Homepage/Homepage";
+import SingleArticle from "./components/SingleArticle/SingleArticle";
+import UserPage from "./components/Userpage/Userpage.jsx";
 
 function App() {
   return (
     <div className="App">
       <Header />
-      <main></main>
+      <main>
```

Comment:
I think you _might_ have an issue using main here. Ordinarily the actual html in public would use the main tag and you can't have more than one. You would get a html error in the terminal if there was a conflict, but to be safe I'd change it to another semantic tag.
If this were plain html you'd be spot on for using it though. 👍